### PR TITLE
feat: Adding contents of memberOf fields with connecting via LDAP

### PR DESCRIPTION
**Pull request by ysirotinskii**
Hi, could you please look at the PR, the fixes include the addition and aggregation of MemberOf elements, which we receive from an external LDAP server

Looking forward to PR approval

### CODE REVIEW

**Comment by hsluoyz** [comment]
@ysirotinskii first pass all CIs, it will not go to review stage before you pass all CIs

File: object/ldap_conn.go [hunk 1]
```diff
@@ -202,7 +202,8 @@ func (l *LdapConn) GetLdapUsers(ldapServer *Ldap) ([]LdapUser, error) {
 			case "postalAddress":
 				user.PostalAddress = attribute.Values[0]
 			case "memberOf":
-				user.MemberOf = attribute.Values[0]
+				fmt.Printf("Synchronized user %s: %v\n", user, attribute.Values)
```
Comment (hsluoyz) [inline object/ldap_conn.go:205]: Don't print to console

File: object/user.go [hunk 2]
```diff
@@ -47,11 +47,12 @@ func InitUserManager() {
...
-	Name        string `xorm:"varchar(100) notnull pk" json:"name"`
-	CreatedTime string `xorm:"varchar(100) index" json:"createdTime"`
-	UpdatedTime string `xorm:"varchar(100)" json:"updatedTime"`
-	DeletedTime string `xorm:"varchar(100)" json:"deletedTime"`
+	Owner       string   `xorm:"varchar(100) notnull pk" json:"owner"`
+	Name        string   `xorm:"varchar(100) notnull pk" json:"name"`
+	CreatedTime string   `xorm:"varchar(100) index" json:"createdTime"`
+	UpdatedTime string   `xorm:"varchar(100)" json:"updatedTime"`
+	DeletedTime string   `xorm:"varchar(100)" json:"deletedTime"`
+	Member      []string `json:"member"`
```
Comment (hsluoyz) [inline object/user.go:55]: You should not add new field. What's member? Is it the groups the user belongs to? We already have groups field

**Review by hsluoyz — CHANGES_REQUESTED** [review]

**Comment by dacongda** [comment]
after research, memberOf seems to be a optional module of ldap and it can be used to find the group via user, I don't think it can be a independence field of user.

**Comment by hsluoyz** [comment]
@ysirotinskii like @dacongda said, the problem you are trying to resolve is meaningful. But your code quality is below the bar. It needs a more skilled person to do the job and make the PR. So closed here for now.

**Pull request closed without merging**